### PR TITLE
feat(emu, bridge): only show relevant binaries - ARM vs nonARM

### PR DESCRIPTION
**Pull request by grdddj**
Connected with https://github.com/trezor/trezor-user-env/issues/147:
- making sure ARM machines will only show ARM emulators/bridges in `dashboard` and vice-versa

### CODE REVIEW

**Comment by vdovhanych** [comment]
would this affect automatic tests when launching bridge or that's done with WebSocket and this shouldn't affect it?

**Comment by grdddj** [comment]
> would this affect automatic tests when launching bridge or that's done with WebSocket and this shouldn't affect it?

It only deals with the dashboard and not showing non-relevant things. As you say, automated tests are fully run through websocket, so it will not affect them.

**Comment by vdovhanych** [comment]
It seems to be working fine :)
Screenshot 2022-03-21 at 17 27 19
I haven't tested the x86 platform yet, but I assume there will be no problems with it either.

**Review by vdovhanych — APPROVED** [review]
lgtm

File: src/binaries.py [hunk 1]
```diff
@@ -27,6 +31,14 @@ def explore_firmwares(args: Any) -> None:
...
     identifier_T1 = "trezor-emu-legacy"
     for fw in glob.glob(firmware_binary_files):
+        # Send only suitable emulators for ARM/non-ARM
+        if IS_ARM and not fw.endswith(ARM_IDENTIFIER):
+            print("On ARM, ignore not-ARM", fw)
+            continue
+        elif not IS_ARM and fw.endswith(ARM_IDENTIFIER):
+            print("Not on ARM, ignore ARM", fw)
+            continue
+
```
[the remark this answers is not in the document]
Comment (grdddj) [reply src/binaries.py]: Changed in [6a7eebd](https://github.com/trezor/trezor-user-env/pull/149/commits/6a7eebd5f10ab789173c45cd661a5114ee45e49c)

File: src/binaries.py [hunk 2]
```diff
@@ -45,23 +57,31 @@ def explore_firmwares(args: Any) -> None:
...
+        "2.0.30-arm64",
+        "2.0.27",
+        "2.0.26",
+        "2.0.19",
+    ]
+
+    if IS_ARM:
+        BRIDGES.extend(bridge for bridge in all_bridges if ARM_IDENTIFIER in bridge)
+    else:
+        BRIDGES.extend(bridge for bridge in all_bridges if ARM_IDENTIFIER not in bridge)
```
[the remark this answers is not in the document]
Comment (grdddj) [reply src/binaries.py]: Right, that is really easier to read. Done in [6a7eebd](https://github.com/trezor/trezor-user-env/pull/149/commits/6a7eebd5f10ab789173c45cd661a5114ee45e49c)

**Review by tsusanka — APPROVED** [review]

**Comment by tsusanka** [comment]
Great, let's squash and merge.

**Pull request merged**